contracts/dao-core/schema/dao-core.json

```diff
@@ -49,7 +49,7 @@
         ]
       },
       "initial_items": {
-        "description": "Initial information for arbitrary contract addresses to be added to the items map. The key is the name of the item in the items map. The value is an enum that either uses an existing address or instantiates a new contract.",
+        "description": "Initial information for arbitrary contract addresses to be added to the items map. The key is the name of the item in the items map. The value is an enum that either uses an existing address or instantiates a new contract.\n\nIt is an error to provide two items with the same key.",
```

Comment:
"It is an error" is a little odd to me. I actually think this whole description could be:

> Initial information for arbitrary contract addresses to be added to the items map. The items map maps the key, a name, to the value, an enum that either uses an existing address or instantiates a new contract.\n\nNote: keys must be unique. Adding an item with an already-existing key will result in an error.